server/src/main/java/org/tctalent/server/service/db/aws/S3ResourceHelper.java

```diff
@@ -295,10 +295,10 @@ public boolean doesBucketExist(String bucket) {
     }
 
     public List<S3ObjectSummary> getObjectSummaries() {
-        List<S3ObjectSummary> objectSummaries = new ArrayList<S3ObjectSummary>();
-        ObjectListing objects = amazonS3.listObjects
+      ObjectListing objects = amazonS3.listObjects
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code uses an outdated API. [ListObjectsV2 is the revised List Objects API, and we recommend you use this revised API for new application developments](https://docs.aws.amazon.com/AmazonS3/latest/API/v2-RESTBucketGET.html).